lib/cisco_node_utils/cmd_ref/radius_server.yaml

```diff
@@ -1,66 +1,101 @@
 # radius_server.yaml
 ---
-_exclude: [ios_xr]
+_template:
+  ios_xr:
+    get_command: "show running-config radius-server"
+    multiple: true
 
 accounting:
+  _exclude: [ios_xr]
   auto_default: false
   kind: boolean
   get_command: "show running-config radius"
-  get_value: '/^radius-server host %s.* (accounting) .*$/'
+  get_value: '/^radius-server host <ip>.* (accounting) .*$/'
   set_value: '<state> radius-server host <ip> accounting'
   default_value: true
 
 acct-port:
-  kind: int
-  get_command: "show running-config radius all"
-  get_value: '/^radius-server host %s.* acct-port (\d+).*$/'
-  set_value: '<state> radius-server host <ip> acct-port <port>'
-  default_value: 1813
+  multiple: true
+  get_value: '/^radius-server host <ip>.* acct-port (\d+).*$/'
+  nexus:
+    default_value: 1813
+    get_command: "show running-config radius all"
```

Comment:
Does it make sense to move this get_command into the `_template`? It's common across most of the Nexus entries.